commands/show-bl-ip.go

```diff
@@ -177,9 +196,17 @@ func showBLIPs(ips []host.AnalysisView, connectedHosts, source bool, delim strin
 	return nil
 }
 
-func showBLIPsHuman(ips []host.AnalysisView, connectedHosts, source bool) error {
+func showBLIPsHuman(ips []blacklist.IPResult, connectedHosts, showNetNames, source bool) error {
 	table := tablewriter.NewWriter(os.Stdout)
-	headers := []string{"IP", "Connections", "Unique Connections", "Total Bytes"}
+	var headers []string
+	if showNetNames {
+		headers = []string{"IP", "Network"}
+	} else {
+		headers = []string{"IP"}
+	}
+
+	headers = append(headers, "Connections", "Unique Connections", "Total Bytes")
```

Comment:
Some places we use "headers" and others we use "headerFields".  Might want to consider keeping them consistent, unless I am missing a fundamental difference between the two?

Reply:
Looks like only this file used `headers`. Changed to `headerFields.`